enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -210,7 +210,8 @@ the container-runtime configuration:
 enable_nri = true
 ```
 
-The plugin is completely optional and is off by default.
+All proposed changes are guarded with a feature gate `mixedCpuAllocation`
```

Comment:
Create a separate Feature Gate section. Also, it starts with a capital, so could be `MixedCPUAllocation`

Reply:
Added on separate section. Thanks